jdaviz/configs/imviz/plugins/coords_info/coords_info.py

```diff
@@ -460,9 +461,15 @@ def _image_viewer_update(self, viewer, x, y):
             self.row3_unreliable = False
 
         maxsize = int(np.ceil(np.log10(np.max(active_layer.layer.shape)))) + 3
-        fmt = 'x={0:0' + str(maxsize) + '.1f} y={1:0' + str(maxsize) + '.1f}'
+        if any(['nan' in map(str, (x, y))]):
+            # don't show nan coordinates:
+            row1a_text = ""
+        else:
+            fmt = 'x={0:0' + str(maxsize) + '.1f} y={1:0' + str(maxsize) + '.1f}'
```

Comment:
Very non-blocker comment!
Is there an opportunity here to use f-string?
```suggestion
            fmt = f'x={{0:0{maxsize}.1f}} y={{1:0{maxsize}.1f}}'
```